CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Create an issue prior to submitting a PR as this can help speed up the process
+* Include tests for your code wherever possible
+* Include any documentation or manual changes where necessary
```

Comment:
`manual`->`man page` I think, manual here sounds like it means by hand (e.g. manual labour)